### DISCUSSION TITLE
Failure to convert AWS QnABot Template

### DISCUSSION THREAD

**Discussion by michaelhoffmantech** [DontShaveTheYak/cf2tf#162] [Q&A] [answered]
I'm trying to convert the AWS QnABot template found here: https://docs.aws.amazon.com/solutions/latest/qnabot-on-aws/aws-cloudformation-template.html. When I run cf2tf, I'm getting an error:

 ```
File "C:\Users\mhoffman\AppData\Local\Programs\Python\Python310\lib\site-packages\cf2tf\conversion\expressions.py", line 421, in get_att
raise ValueError(
ValueError: Could not convert Cloudformation property QnaIndex to Terraform attribute of ['id', 'certificate_validation_records', 'dns_target'].
```

I'm still somewhat new to IaC so my apologies for not being able to provide more details.

**Comment by shadycuz** [comment, marked as answer]
Just a bug, an issue would be a better place for it.

Will look at it. Thanks for posting.

**Comment by michaelhoffmantech** [comment]
Thanks, no rush as I'm still in research mode for a few weeks.
